src/main/java/net/hypixel/nerdbot/generator/item/overlay/OverlayType.java

```diff
@@ -0,0 +1,52 @@
+package net.hypixel.nerdbot.generator.item.overlay;
+
+import java.awt.image.BufferedImage;
+import java.util.HashMap;
+
+public enum OverlayType {
+    NORMAL, MAPPED, DUAL_LAYER;
```

Comment:
Put them all on a separate line because this'll probably happen when someone formats the class either way